core-web/libs/template-builder/src/lib/components/template-builder/store/template-builder.store.spec.ts

```diff
@@ -270,30 +287,23 @@ describe('DotTemplateBuilderStore', () => {
             containerMap: {}
         });
 
+        const newWidth = 2;
         const affectedColumns: DotGridStackNode[] = [
             {
-                x: 1,
-                y: 0,
-                w: 1,
-                id: firstId
-            },
-            { x: 0, y: 0, w: 1, id: secondId },
-            { x: 3, y: 0, w: 1, id: thirdId },
-            { x: 4, y: 0, w: 5, id: fourthId }
+                ...firstBox,
+                w: newWidth
+            }
         ];
-        const createdWidgets = affectedColumns.map((column) => ({
-            ...column,
-            parentId
-        }));
 
         service.updateColumnGridStackData(affectedColumns);
 
         items$.subscribe((items) => {
             const row = items.find((item) => item.id === parentId);
-            expect(row?.subGridOpts?.children).toEqual(createdWidgets);
+            expect(row.subGridOpts.children[0].w).toEqual(newWidth);
+            done();
         });
     });
-    it('should update styleClass data of a column', () => {
+    it('should update styleClass data of a column', (done) => {
         expect.assertions(1);
```

Comment:
I'm not sure, but I think that every `expect.assertions(n)` is basically unnecessary, it's not doing anything. We can test it and if it's not needed we can delete them